scripts/upgrade

```diff
@@ -175,23 +177,12 @@ ynh_add_systemd_config --service=$service_name
 
 #=================================================
 # GENERIC FINALIZATION
-#=================================================
-# SECURE FILES AND DIRECTORIES
-#=================================================
-ynh_script_progression --message="Securing files and directories..."
-
-# Set permissions on app files
-chown -R $app:$app ${final_path}/
-chmod -R 0644 ${final_path}/*
-find ${final_path}/ -type d -exec chmod +x {} \;
-chmod o-rwx "${final_path}"
-
```

Comment:
Uuuh, but why not re-inforce permissions during upgrade ?

I suspect there are many existing setups which don't match the permissions that are supposed to be there from the new install script ?

Reply:
Forgotten to include it at DOWNLOAD, CHECK AND UNPACK SOURCE step, now done